scripts/cross-compile.sh

```diff
@@ -14,8 +14,8 @@ fi
 for platform in linux-amd64 linux-arm64 linux-ppc64le linux-s390x darwin-amd64 windows-amd64 ; do
   echo "Cross compiling $platform and placing binary at dist/bin/$platform/"
   if [ $platform == "windows-amd64" ]; then
-    GOARCH=amd64 GOOS=windows go build -o dist/bin/$platform/odo.exe -ldflags="$COMMON_FLAGS" ./cmd/odo/
+    GOARCH=amd64 GOOS=windows go build -o dist/bin/$platform/odo.exe -ldflags="-s -w $COMMON_FLAGS" ./cmd/odo/
```

Comment:
why not add to $COMMON_FLAGS in makefile?

Reply:
Do we want to make stripped binaries every time or allow developer to still have access to the flags with `make bin`?

Reply:
ok, good point